frontend-service-client/src/main/java/com/amazonaws/blox/model/ResourceNotFoundException.java

```diff
@@ -0,0 +1,85 @@
+/**
+
+*/
+package com.amazonaws.blox.model;
+
+import javax.annotation.Generated;
+
+/**
+ * 
+ */
+@Generated("com.amazonaws:aws-java-sdk-code-generator")
+public class ResourceNotFoundException extends com.amazonaws.blox.model.BloxException {
```

Comment:
I'm kind-of surprised that this worked; how did the generator come up with this (correct) name, but the name of the object in the Swagger model and in the Frontend is ResourceNotFoundException**Response**?

If there's some trick to it, it might be worth documenting somewhere for when we add more exceptions.

Reply:
There is a configuration while generating the SDK, see ` "exceptionFor404=ResourceNotFoundException"`  (https://github.com/blox/blox/pull/365/files#diff-c197962302397baf3a4cc36463dce5eaR198)

I'm kind of glad to see apigateway leveraging the exception model we define in the swagger.

Reply:
Right, but that is referencing `ResourceNotFoundException`, right? How does it know to use the properties from the `ResourceNotFoundExceptionResponse` definition in the swagger?

Reply:
I think it's the interesting part of apigateway sdk generation.   It respects the exception model, but don't respect the exception name in the swagger. If not provided the  "exceptionFor404=ResourceNotFoundException", the default exception `NotFoundException` will be generated and with right data model as well.

Reply:
Ah, I see, makes sense. Probably worth adding a comment to that effect here.